Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -39,7 +39,7 @@ public extension String {
 	/// SwifterSwift: Array of characters of a string.
 	///
 	public var charactersArray: [Character] {
-		return Array(characters)
+		return Array(self)
```

Comment:
Do we get an array of characters here or an array of a Single string?

Reply:
Guess I have to read about the new Strings 🤔 Implicit cast?

Reply:
It returns an array of characters :)

Reply:
I think is because string now has collection conformance again, so it uses Array(s: Sequence) and returns a sequence of its characters. Did this test on playground
<img width="717" alt="screen shot 2017-11-05 at 10 57 27" src="https://user-images.githubusercontent.com/8292651/32414996-f2f97250-c218-11e7-8ca9-df8ff4213dc9.png">

Reply:
Can we do this? Forgive me I haven't had to deal with this yet 😃 But we may want to re-evaluate all the string extensions in general and see if some are obsolete
```Swift
var chars: [Characters] = "Swift"
```

Reply:
I think not, swift compiler complains about cannot convert String to [Character], 
<img width="620" alt="screen shot 2017-11-05 at 11 31 13" src="https://user-images.githubusercontent.com/8292651/32415205-022f0bdc-c21d-11e7-97a6-5ebe75dd6d24.png">

Reply:
@LucianoPAlmeida Appreciate you testing that! It's very hard for me to run a playground on my slow iMac 😭